dags/myla311/dag-311-to-arc.py

```diff
@@ -0,0 +1,96 @@
+import pandas as pd
+import arcgis 
+from arcgis.gis import GIS
+from arcgis.features import FeatureLayerCollection
+import geopandas as gpd 
+import os 
+import shutil
+import pathlib
+
+from airflow import DAG
+from airflow.operators.bash_operator import BashOperator
+from airflow.operators.python_operator import PythonOperator
+from datetime import datetime, timedelta
+
+def overwrite_layer(**kwargs):
+    """
+    Overwrites the layer 
+    on a ESRI GIS
+    """
+    print('logging into ESRI')
+
+    gis = GIS("http://lahub.maps.arcgis.com/home/index.html", 
+            os.environ.get('LAHUB_USERNAME'),
+            os.environ.get('LAHUB_PASSWORD'))
+    # get the item 
+
+    my_content = gis.content.search(query="311_test owner:" + gis.users.me.username, 
+                                item_type="Feature Layer", 
+                                max_items=15)
+    item=my_content[0]
+    print(f'item is {item}')
+    print("overwriting")
+    # overwrite the file 
+    feature_layer_collection = FeatureLayerCollection.fromitem(item)
+    large_path = '/tmp/service_requests.zip'
+    print(feature_layer_collection.manager.overwrite(large_path))
+    return True
+
+default_args = {
+    'owner': 'hunterowens',
+    'depends_on_past': False,
+    'start_date': datetime(2020, 1, 10),
+    'email': ['ITAData@lacity.org'],
+    'email_on_failure': False,
+    'email_on_retry': False,
+    'retries': 1,
+    'retry_delay': timedelta(minutes=5),
+    # 'queue': 'bash_queue',
+    # 'pool': 'backfill',
+    # 'priority_weight': 10,
+    # 'end_date': datetime(2016, 1, 1),
+}
+
+dag = DAG('311-sync', default_args=default_args, schedule_interval=timedelta(days=1))
+
+t1 = BashOperator(
+    task_id='download_date',
+    bash_command='',
+    dag=dag)
+
+t2 = BashOperator(

```

Comment:
This seems like overkill to me.

Reply:
??

Reply:
By this I meant having a separate operator for each `bash` command.